ui/src/components/Status/__tests__/Status.test.ts

```diff
@@ -0,0 +1,47 @@
+import { shallowMount } from "@vue/test-utils";
+import Status from '../Status.vue';
+import { assert } from "chai";
+
+describe("Status test scripts", () => {
+    test("Renders correctly", () => {
```

Comment:
Do you mind using `it` instead? Just to make things more consistent.

Also I think that we want to slowly but surely migrate from chai to the expect-based-api which is already part of jest. Sorry that I did not mention that to you earlier. Otherwise it looks fine to me.

Reply:
cool i ll do this. but i also read somewhere that "it" is an alias of "test" and that test is the official api for jest.

Reply:
> cool i ll do this. but i also read somewhere that "it" is an alias of "test" and that test is the official api for jest.

:D I don't care so much about test vs it. Maybe I should not have mentioned it then... d'oh. To me personally, this chai thing is more important. :D

it/test: Yeah the jest docs talks about test much more and `it` is only a side note. They only state it is an alias. So I guess `test` would be even better than `it`... Maybe we can just have a quick chat about it? I think in the mid-long-term we should be consistent and go with the main-stream => `test`. I mean renaming `it` to `test` is just a find-and-replace-operation in the end...